algorithms/tree/segment_tree/SegmentTree.py

```diff
@@ -0,0 +1,66 @@
+class SegmentTree:
+    def __init__(self, values):
+        self.valarr = values
+        self.arr = dict()
+
+    #start is the starting index of node.
+    #end is the ending index of node.
+    #l is the lower bound of given query.
+    #r is the upper bound of given query.
+
+    def buildTree(self, start, end, node):
```

Comment:
could you make the function, variable and file names snake case (ex `build_tree`)?

Reply:
Functions, variables, classes and files name changed to snake case instead of camel case.